webapp/src/views/ServiceList.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <div class="container">
+    <h1>Service List</h1>
```

Comment:
```suggestion
    <h1>Dienstzuteilung</h1>
```

As long as we don't have localization, keep the UI in German.